pkg/cmd/admin/router/router.go

```diff
@@ -156,6 +156,13 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 		glog.Fatal(err)
 	}
 
+	ports = append(ports, kapi.ContainerPort{
```

Comment:
updated.  The template already looks for > 0 so this works nicely to turn off stats completely.